riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks:
+- discussion
+deployments:
+  static:
+    parameters:
+      cacheControl:
+      - pattern: .*\.js$
+        value: public, max-age=315360000
+      - pattern: .*\.map$
+        value: public, max-age=315360000
+      - pattern: .*
+        value: public, max-age=60
+      mimeTypes:
+        js: application/javascript
+        json: application/json
+        map: text/plain
+      prefixStack: false
```

Comment:
Yeah the reason I removed them is because you get warning messages in riff-raff when you deploy with defaults that you don't need.
![image](https://user-images.githubusercontent.com/14179210/27825643-1a81fa2c-60a8-11e7-8e04-30a651ac5673.png)